containerfiles/initrd-builder/osbuilder/dracut/dracut-kata-cc.conf

```diff
@@ -0,0 +1,4 @@
+# SNP
```

Comment:
There is a lot of commonality between the various .conf files.

Couldn't we instead have fragments that we'd put in the existing dracut.conf.d, in order to avoid copy-paste inheritance and future maintenance burden?

Something like 20-dracut-cc-snp.conf, 21-dracut-cc-tdx.conf, 25-dracut-nvidia.conf, and so on?